TranformingData/transformingDataExample.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3.6
+
+import dask.dataframe as dd
+import pandas as pd
+import sys
+import pickle
+import matplotlib.pyplot as plt
+
+def dataFrameNames():
```

Comment:
Why are you using a function for this?

Wouldn't global variable be enough?

Reply:
https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/8bc76cf6252e9c9880dde9f18f530bee3f2c9c38

Reply:
There was a reason that I truly don't remember.
https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/c6d4263dfb9ce1c0335812bff7ab0f0074c53b53

Reply:
I've remembered. I was using it in another file (in another script) and imported names as function cause I cant import variable ;) This not make sense in this case.